tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -306,3 +308,38 @@ Re-validate License For Disabled Application From Enabled Page
    Wait Until Page Contains   To remove card click
    ${buttons_here}=  Get WebElements    xpath://div[contains(@class,'popover__body')]//button[text()='here']
    Click Element  ${buttons_here}[0]
+
+Check CSS Property Has The Expected Value
+    [Documentation]     Compare the actual CSS property value with the expected one
+    [Arguments]   ${locator}    ${property}    ${exp_value}   ${operation}=equal
+    ${el_text}=   Get Text   xpath:${locator}
+    Log    Text of the target element: ${el_text}
+    ${actual_value}=    Get CSS Property Value   xpath:${locator}    ${property}
+    IF    $operation == "contains"
+        Run Keyword And Continue On Failure   Should Contain    ${actual_value}    ${exp_value}
+    ELSE
+        Run Keyword And Continue On Failure   Should Be Equal    ${actual_value}    ${exp_value}
+    END
+
+Check CSS Style Is The Expected One
```

Comment:
Could be renamed to `Verify JupyterHub Card CSS Style`. Not sure if it should be defined in this resource file or in the test itself

Reply:
hm I will move to the test file. Will think about the name.

Reply:
done